app/views/registration_mailer/confirmation_email.txt.erb

```diff
@@ -0,0 +1,6 @@
+Hello <%= @user.first_name %>!
```

Comment:
raw text version for email platforms that prefer it

Reply:
probably internet explorer...